ironmq/src/test/scala/akka/stream/alpakka/ironmq/AkkaFixture.scala

```diff
@@ -18,7 +27,12 @@ trait AkkaFixture extends ConfigFixture with BeforeAndAfterEach { _: Suite =>
   }
 
   override protected def afterEach(): Unit = {
-    actorSystem.terminate()
+    Await.result(actorSystem.terminate(), actorSystemTerminateTimeout)
     super.afterEach()
   }
 }
+
+object AkkaFixture {
+  val DefaultActorSystemTerminateTimeout: Duration = 10.seconds
```

Comment:
OK, though would be good to at some point make sure all Alpakka tests use the same style here... not neccessary in this PR though, is good for now